### PR TITLE
fix: zero chat state

### PR DESCRIPTION
**Pull request by suprko**
### PR Checklist
- [x] I have merged `dev` into my branch before submitting
- [x] I have checked with either the repo owner or person who previously worked on this feature to ensure correctness (if applicable)
- [x] I have ensured that I am not using any English strings, only `t('translation_keys)`
- [x] The pull request name follows the standard commit guidelines, e.g. `feat(files): filter by file type`

### Relevant user stories or bugs
Very quick fix to prevent users who have at least one chat/channel from seeing the zero chat screen, if they manage to deselect the current `activeChat`.

### Test plan
Tested by @ebilg who wants to merge so that this can be tested alongside another bugfix
https://peerio.tpondemand.com/entity/7262

------

The repo owner is @anri-asaturov. The responsible QA specialist is @ebilg.

⬇️ **Note to reviewer:** please squash when merging.

### CODE REVIEW

**Comment by suprko** [comment]
Sorry waiting for testing

**Comment by suprko** [comment]
@anri-asaturov testing finished, OK to merge now if it looks good to you

File: src/ui/chat/components/NoChatSelected.jsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ class NoChatSelected extends React.Component {
     };
 
     render() {
-        if (chatStore.loading) return null;
+        if (chatStore.loading || chatStore.chats.length || chatStore.channels.length) return null;
```
Comment (anri-asaturov) [inline src/ui/chat/components/NoChatSelected.jsx]: `chatStore.chats` contains both chnnels and DMs
`chatStore.channels` and `chatStore.directMessages` are computed values based on `chatStore.chats`

Comment (suprko) [reply src/ui/chat/components/NoChatSelected.jsx]: Ah OK. Updated to just use `chatStore.chats.length`

**Pull request merged**